Units/parser-typescript.r/ts-const.d/input.ts

```diff
@@ -0,0 +1,69 @@
+import * as three from "three";
```

Comment:
I would like you to write the source of the test input file like
```
// Taken from issue #2488 submitted by @sparkcanon
```

(Fix the example if my English is wrong.)

Reply:
Good point, thanks! I've updated the test file.

Reply:
Updated in c8cb4a4 ?
As far as seeing https://github.com/universal-ctags/ctags/pull/2493/files, the test input is not changed.

Reply:
yeah, I was doing `git commit --amend` and forgot to run `git add` first. It should be ok now.